src/components/TeacherViews/LessonOverview/LessonOverview.view.tsx

```diff
@@ -46,6 +47,17 @@ export type LessonOverviewProps = {
     | SpecialistLessonOverview;
 };
 
+// helper function to dedupe key learning points from the header in legacy lessons
+export const isPupilLessonOutcomeInKeyLearningPoints = (
```

Comment:
If this helper were changed to return `pupilLessonOutcome` if it is found, or `undefined` if it is not, it would make the view a bit simpler